content/_guides/clients.md

```diff
@@ -45,6 +45,10 @@ commonly used by Android users, and
 [LimeChat](https://apps.apple.com/ch/app/limechat-irc-client/id298766460) is
 popular with iOS users.
 
+For a self-hosted, web-based client, [thelounge](https://thelounge.chat/) is
+a popular option, and comes with a built-in bouncer. It does require more
+configuration and maintenance than other bouncer clients, however.
+
```

Comment:
```suggestion
For a self-hosted, web-based client, [The Lounge](https://thelounge.chat/) is
a popular option. It acts as a bouncer and works great on phones.
```

Lounge doesn't have a bouncer (it just works like one due to being always connected). Unsure how it requires more configuration compared to something like znc (nor why does it need to be mentioned here).